examples/phantom-parts/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en">
+<head>
+    <title>Phantom Parts</title>
+
+    <!-- Live reload script -->
+    <script src="http://localhost:35729/livereload.js?snipver=1"></script>
+
+    <script type="module" src="main.js"></script>
+
+    <link
+            rel="stylesheet"
+            href="https://unpkg.com/@vertexvis/viewer@0.17.x/dist/viewer/viewer.css"
```

Comment:
We might need to publish a new version in order for the example to work for external users.